CHANGELOG.md

```diff
@@ -8,36 +8,47 @@
 > * [CHANGELOG_1.2x.md](./documentation/changelog/CHANGELOG_1.2x.md) - v1.20.0 to v1.29.2
 
 # UNRELEASED
-- fix(cli): handle disabled payment channel manager gracefully in lotus info command ([filecoin-project/lotus#13198](https://github.com/filecoin-project/lotus/pull/13198))
-- chore(deps): update go-libp2p to v0.42.0 ([filecoin-project/lotus#13190](https://github.com/filecoin-project/lotus/pull/13190))
-- chore: update Go version to 1.23.10 ([filecoin-project/lotus#13190](https://github.com/filecoin-project/lotus/pull/13190))
-- chore(deps): upgrade boxo to v0.32.0 and go-ipld-cbor to v0.2.1 ([filecoin-project/lotus#13202](https://github.com/filecoin-project/lotus/pull/13202))
+
+# Node v1.33.1-rc1 / 2025-07-18
+This is the first release candidate of Lotus v1.33.1, whic includes..
+
+## ☢️ Upgrade Warnings ☢️
+- This release requires a minimum Go version of v1.23.10 or higher??
```

Comment:
1.23 should be fine I think, the listing of 1.23.10 in go.mod now tells Go to go and fetch the 1.23.10 stdlib to build so even if you build with an older Go it'll go and fetch the newer one. So I think you can actually just leave this warning off because 1.33.0 was 1.23 minimum already anyway.

Reply:
Okay, that is great! I will remove this warning then